app/listeners/casino/login_credential_acceptor_listener.rb

```diff
@@ -16,16 +18,19 @@ def two_factor_authentication_pending(ticket_granting_ticket)
   end
 
   def invalid_login_credentials(login_ticket)
-    @controller.flash.now[:error] = I18n.t('login_credential_acceptor.invalid_login_credentials')
+    Rails.logger.error I18n.t('login_credential_acceptor.invalid_login_credentials')
```

Comment:
Uhh... Error logging in the current locale? I pray we do not deploy in China before we fix this :sweat_smile: 

_(it may be done any time later, not now)_